python/ray/test_utils.py

```diff
@@ -542,3 +542,20 @@ def load_test_config(config_file_name):
 def set_setup_func():
     import ray._private.runtime_env as runtime_env
     runtime_env.VAR = "hello world"
+
+
+def get_nightly_url():
+    """Returns the Ray nightly URL."""
+    ray_version = ray.__version__
+    python_version = f"{sys.version_info.major}{sys.version_info.minor}"
+    os_strings = {
+        "darwin": "macosx_10_13_intel",
+        "linux": "manylinux2014_x86_64",
+        "win32": "win_amd64"
```

Comment:
This is actually a bit more complex ⭕
If `py38`, then:
```
darwin = macosx_10_13_x86_64
```

Reply:
Ah thanks for catching this!